misc/build/build-fedora.sh

```diff
@@ -0,0 +1,84 @@
+export BUILD_TYPE='Release'
+export SDL_VERSION='release-2.30.0'
+export OPENCV_VERSION='4.9.0'
+export CATCH_VERSION='v3.5.2'
+export SPDLOG_VERSION='v1.13.0'
+export EXIV2_VERSION='v0.28.1'
+export GENERATOR='Ninja Multi-Config'
+
+git submodule update --init
+
+
+git clone https://github.com/catchorg/Catch2.git catch --depth 1 --branch $CATCH_VERSION
+cd catch
+cmake -B build \
+  -DCMAKE_C_COMPILER=gcc \
```

Comment:
If you'd like you can make this a var, e.g. C_COMPILER and reuse it everywhere in the script, also CXX_COMPILER, so that it would be easy to switch the compiler at the top